examples/guides/mp-restful-webservice/README.adoc

```diff
@@ -669,223 +499,210 @@ accepting it. But omitting that validation lets us create an easy health check
 that we can use by simply setting the greeting to blank from
 a `curl` command.
 
-=== Add a new JAX-RS resource class
-Create `HealthResource.java` to define the endpoints for checking if the service is
+Helidon MicroProfile provides a built-in health framework. We can tap into that
```

Comment:
"tap into that" ---> "use that framework"

Reply:
done